test/integration/index.spec.ts

```diff
@@ -1426,75 +1425,6 @@ describe('Worker', () => {
 		await expect(ctx.flush(ctx.session)).rejects.toThrowError();
 	});
 
-	it('should not re-enqueue requests after execute failure', async () => {
-		const typeContract = ctx.worker.typeContracts['card@1.0.0'];
-		assert(typeContract);
-
-		await ctx.worker.insertCard(
-			ctx.logContext,
-			ctx.session,
-			ctx.worker.typeContracts['action-request@1.0.0'],
-			{},
-			{
-				data: {
-					action: 'action-create-card@1.0.0',
-					context: ctx.logContext,
-					card: typeContract.id,
-					type: typeContract.type,
-					actor: ctx.adminUserId,
-					epoch: new Date().valueOf(),
-					input: {
-						id: typeContract.id,
-					},
-					timestamp: new Date().toISOString(),
-					arguments: {
-						reason: null,
-						properties: {
-							slug: 'foo',
-							version: '1.0.0',
-							data: {
-								foo: 'bar',
-							},
-						},
-					},
-				},
-			},
-		);
-
-		const enqueuedRequest1 = await ctx.dequeue();
-		assert(enqueuedRequest1);
-
-		await ctx.kernel.insertContract(ctx.logContext, ctx.session, {
-			slug: 'foo',
-			type: 'card@1.0.0',
-			version: '1.0.0',
-			data: {
-				foo: 'bar',
-			},
-		});
-
-		await ctx.worker.consumer.postResults(
-			autumndbTestUtils.generateRandomId(),
-			ctx.logContext,
-			enqueuedRequest1 as any,
-			{
-				error: false,
-				data: {
-					id: autumndbTestUtils.generateRandomId(),
-					type: 'card@1.0.0',
-					slug: 'foo',
-				},
-			},
-		);
-
-		await expect(
-			ctx.worker.execute(ctx.session, enqueuedRequest1),
```

Comment:
I *think* this expects a "Duplicate" error to be thrown from either https://github.com/product-os/jellyfish-worker/blob/e9e5145c92dee74f915f95efe3ae24b1a475d42d/lib/queue/events.ts#L106 or https://github.com/product-os/jellyfish-worker/blob/e9e5145c92dee74f915f95efe3ae24b1a475d42d/lib/queue/consumer.ts#L77

I think, given this PR, that here should be an `expect` for `errors.WorkerInvalidActionRequest("action-request already executed: ${request.id}`")